nextflow_schema.json

```diff
@@ -368,6 +368,13 @@
                     "type": "integer",
                     "default": 110,
                     "description": "VEP cache version"
+                },
+                "deepvariant_model_type": {
+                    "type": "string",
+                    "default": "PACBIO",
+                    "description": "Sets the model type for DeepVariant",
+                    "hidden": true,
+                    "enum": ["WGS", "WES", "PACBIO", "ONT_R104", "HYBRID_PACBIO_ILLUMINA"]
```

Comment:
Would you ever set anything but `PACBIO` or `ONT_R104`? I understand it as WGS and WES being optimised for Short read data

Reply:
No not really, I was just in the mood for giving people freedom.